lib/components/chat_panel.dart

```diff
@@ -54,35 +80,96 @@ class _ChatPanelWidgetState extends State<ChatPanelWidget> {
                         context: context,
                         builder: (context) {
                           return Dialog(
-                            child: ListView(shrinkWrap: true, children: [
-                              ListTile(
-                                  title: Text('Delete Message'),
-                                  onTap: () {
-                                    final model = Provider.of<UserModel>(
-                                        context,
-                                        listen: false);
-                                    model.delete(model.channels.first,
-                                        message.messageId);
-                                    Navigator.pop(context);
-                                  }),
-                              ListTile(
-                                  title: Text('Timeout ${message.author}'),
-                                  onTap: () {}),
-                              ListTile(
-                                  title: Text('Ban ${message.author}'),
-                                  onTap: () {}),
-                              ListTile(
-                                  title: Text('Unban ${message.author}'),
-                                  onTap: () {
-                                    final model = Provider.of<UserModel>(
-                                        context,
-                                        listen: false);
-                                    model.unban(
-                                        model.channels.first, message.author);
-                                    Navigator.pop(context);
-                                  }),
-                            ]),
-                          );
+                              shape: RoundedRectangleBorder(
```

Comment:
Leave out custom shapes, the material defaults usually look the most consistent.